content/en/tracing/trace_collection/library_config/go.md

```diff
@@ -118,20 +118,26 @@ Datadog may collect [environmental and diagnostic information about your system]
 
 The [APM environment name][7] may be configured [in the Agent][8] or using the [WithEnv][3] start option of the tracer.
 
-## B3 headers extraction and injection
+## Trace context propagation for distributed tracing
 
-The Datadog APM tracer supports [B3 headers extraction][9] and injection for distributed tracing.
+The Datadog APM tracer supports [B3 headers][9], [W3C][14] extraction and injection for distributed tracing.
```

Comment:
```suggestion
The Datadog APM tracer supports [B3 headers][9], [W3C][14] extraction, and injection for distributed tracing.
```
The correct grammar here depends on what the items are. My edit assumes that the tracer supports:
- B3 headers
- W3C extraction
- injection for distributed tracing

If the words are arranged or grouped differently, let me know and can give you the appropriate punctuation. Or to make it perfectly clear, you can change this sentence to use a bulleted list.

Reply:
Maybe it's two items? In that case, it should be:

The Datadog APM tracer supports [B3 headers][9] and [W3C][14] extraction and injection for distributed tracing.

In that case, the bulleted list would be clearer.

The Datadog APM tracer supports the following for distributed tracing:
- [B3 headers][9]
- [W3C][14] extraction and injection

Reply:
We support both injection and extraction of both B3 headers and W3C headers. So probably can just be:

"The Datadog APM tracer supports extraction and injection of [B3][9] and [W3C][14] headers for distributed tracing."

Reply:
Thank you, Katie. That's perfect.